pro/pro-setup.rst

```diff
@@ -8,7 +8,7 @@ This guide covers how to get setup with Netmaker Professional. For pricing,visit
 Get a License
 =================================
 
-Visit `<https://app.netmaker.io>`_ to set up a self-hosted instance on the Netmaker SAAS platform. When you first visit the site, You should be prompted to create an account. After you make one, you will be automatically directed to a free SAAS instance. You will need to delete this free tier before creating a self-hosted tenant. click on the manage account button located on the side bar.
+Visit `<https://app.netmaker.io>`_ to set up a self-hosted instance on the Netmaker SAAS platform. When you first visit the site, You should be prompted to create an account. After you make one, you will be automatically directed to a free SAAS instance. You will need to go to account managemement. Click on the manage account button located on the side bar.
```

Comment:
mention about clicking on the `switch tenant tab` , a screenshot would help here

Reply:
done